internal/receiver/databricksreceiver/documentation.md

```diff
@@ -8,18 +8,175 @@ These are the metrics available for this scraper.
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.backend.daemon.driver.dataplaneeventlistener** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.backend.daemon.driver.dbceventlogginglistener** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.photon.photoncleanuplistener** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.spark.util.executortimelogginglistener** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.spark.util.usagelogginglistener** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.sql.advice.advisorlistener** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.sql.debugger.querywatchdoglistener** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.sql.execution.ui.iocachelistener** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.listenerprocessingtime.databricks.sql.io.caching.repeatedreadsestimator** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.queue.appstatus.listenerprocessingtime** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.queue.executormanagement.listenerprocessingtime** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.queue.shared.listenerprocessingtime** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.spark.timer.livelistenerbus.queue.streams.listenerprocessingtime** | n/a | ms | Sum(Double) | <ul> <li>cluster_id</li> <li>spark_app_id</li> </ul> |
+| **databricks.tasks.run.duration** | The execution duration in milliseconds per completed task | ms | Gauge(Int) | <ul> <li>job_id</li> <li>task_id</li> </ul> |
+| **databricks.tasks.schedule.status** | A snapshot of the pause/run status per task taken at each scrape 0=PAUSED, 1=UNPAUSED, 2=NOT_SCHEDULED | {status} | Gauge(Int) | <ul> <li>job_id</li> <li>task_id</li> <li>task_type</li> </ul> |
+
+**Highlighted metrics** are emitted by default. Other metrics are optional and not emitted by default.
```

Comment:
Definitely not a databricks user but this seems like a lot of potentially high cardinality default metrics?

Reply:
All metrics are default for now until we can get more information about what folks find useful. I haven't seen high cardinality but I've only run a few tutorial workloads so I suppose that's to be expected. Do you have a remedy in mind?